data/presets/highway/path/_crossing.json

```diff
@@ -0,0 +1,30 @@
+{
+    "icon": "temaki-cyclist_crosswalk",
+    "fields": [
+        "crossing",
+        "{@templates/crossing/markings}",
+        "{@templates/crossing/defaults}",
+        "surface"
+    ],
+    "moreFields": [
+        "{@templates/crossing/geometry_way_more}",
+        "{@templates/crossing/bicycle_more}",
+        "flashing_lights"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "path": "crossing"
+    },
+    "addTags": {
+        "highway": "path",
+        "path": "crossing"
+    },
+    "reference": {
+        "key": "path",
+        "value": "crossing"
+    },
+    "searchable": false,
+    "name": "Cycle & Foot Crossing"
```

Comment:
Thinking about this again: Is this really the right name? It is not wrong, to describe a path as Bike+Foot infrastructure. However this wording was inspired by https://github.com/openstreetmap/id-tagging-schema/pull/1201/files#diff-3b8f616654b5d2845f63eb05ce32b27741d3945ad0a71e55033de12a656d8b56R48 which behaves differently from this preset because it also has the `addTags` part about `foot+bicycle=designated`.

Since the preset is unsearchable, no one would be confused by this when searching for presets.

And also, I thing that even if people do not add the explicit `access` keys, routers would still consider the generic `path` bike + foot infrastructure (which is why some regions prefer this tagging for the bike+foot-case in the first place).

So my opinion is this is OK … or can be changed in follow up PRs when other cases are added or reevaluated.